crates/rv/src/main.rs

```diff
@@ -296,6 +299,7 @@ async fn run() -> Result<()> {
                 CacheCommand::Clean => cache_clean(&config)?,
                 CacheCommand::Prune => cache_prune(&config)?,
             },
+            Commands::Selfupdate => selfupdate().await.unwrap(), // FIXME: don't unwrap
```

Comment:
The basic functionality is in place, but I couldn't figure out how to coerce a `miette::Report` into an `Error` (from `crates/rv/src/main.rs`) here.